src/styles/global.ts

```diff
@@ -0,0 +1,25 @@
+import { createGlobalStyle, DefaultTheme } from "styled-components";
+
```

Comment:
I would probably consider using css variables for the theming implementation, however this approach is also fine.

Reply:
I haven't used css variables in the past, but I will give them a look for sure.